proposals/dao/fip_33.ts

```diff
@@ -0,0 +1,218 @@
+import { ethers } from 'hardhat';
+import chai, { expect } from 'chai';
+import CBN from 'chai-bn';
+import {
+  DeployUpgradeFunc,
+  NamedContracts,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  ValidateUpgradeFunc
+} from '../../types/types';
+import { TransactionResponse } from '@ethersproject/providers';
+
+before(async () => {
+  chai.use(CBN(ethers.BigNumber));
+});
+
+// Constants
+// CR oracle wrapper
+const CR_KEEPER_INCENTIVE = ethers.constants.WeiPerEther.mul(1000); // 1000 FEI
+
+// CR oracle guardian
+const CR_GUARDIAN_FREQUENCY = 12 * 60 * 60; // 12 hours
```

Comment:
nit but shouldn't this be period instead of frequency?

Reply:
shit